pyne/r2s.py

```diff
@@ -126,6 +127,10 @@ def irradiation_setup(flux_mesh, cell_mats, cell_fracs, alara_params,
         List of the decay times. If no decay times given, use '1 s'.
     sub_voxel : bool, optional
         If true, sub-voxel r2s work flow  will be used.
+    responses : list of str, optional
+        The list of requested responses.
+    wdr_file : str
+        File name of the standard to calculate wdr.
```

Comment:
just the filename, or also the path?

Reply:
I should be the path. I will update the doc here.